src/ert/config/field.py

```diff
@@ -166,7 +167,9 @@ def write_to_runpath(
         self, run_path: Path, real_nr: int, ensemble: Ensemble
     ) -> None:
         t = time.perf_counter()
-        file_out = run_path.joinpath(self.output_file)
+        file_out = run_path.joinpath(
+            substitute_runpath_name(str(self.output_file), real_nr, ensemble.iteration)
```

Comment:
why `str` here and not other places? 🤔

Reply:
This is the only class that uses `Path` for `self.output_file`